extensions/wikia/PremiumPageHeader/styles/_banner-notifications-hack.scss

```diff
@@ -4,6 +4,7 @@
 .banner-notifications-wrapper {
 	position: relative;
 	z-index: $zTop;
+	top: 0 !important;
```

Comment:
We will have to test it properly, however, this is causing the issue https://github.com/Wikia/app/blob/cbfe441e7b83b2305c29efade30444ee94bb7e76/extensions/wikia/BannerNotifications/js/BannerNotifications.js#L231